.github/workflows/ci.yml

```diff
@@ -118,7 +118,7 @@ jobs:
         fi
 
     - name: Submit code coverage
-      if: success() && matrix.php-version == '7.4'
```

Comment:
I don't have a problem with these changes, but wondering what this fixes.

Reply:
Is it so the patch codecov % isn't way off for a single failure?

Reply:
Now I'm thinking the success() is just redundant.

Reply:
> Now I'm thinking the success() is just redundant.

Yes, as stated in the commit message.